build.cake

```diff
@@ -578,7 +643,7 @@ void BuildProject(string projectPath, string configuration, MSBuildPlatform buil
     }
 }
  
-void RunTest(FilePath exePath, DirectoryPath workingDir)
+void RunTest(FilePath exePath, DirectoryPath workingDir, string framework, ref List<string> errorDetail)
```

Comment:
Why does the list need to be a `ref`, you are not assigning to it, just adding to it? Objects are already passed by reference.